lang/src/idl.rs

```diff
@@ -48,6 +48,7 @@ pub type IdlCreateAccounts<'info> = crate::ctor::Ctor<'info>;
 #[derive(Accounts)]
 pub struct IdlAccounts<'info> {
     #[account(mut, has_one = authority)]
+    #[allow(deprecated)]
     pub idl: ProgramAccount<'info, IdlAccount>,
```

Comment:
Not sure how to replace `ProgramAccount` for `IdlAccount`. We can't use `Box<Account<'info, IdlAccount>>` because `IdlAccount` do not implement `Owner` trait.